### PR TITLE
MWPW-137137 - Fix styling of Text block legal variant

### PR DESCRIPTION
**Pull request by Sartxi**
There was a styling conflict introduced in the text block legal variant by the body content styling fallback that was added in our shared `decorateBlockText` function. I also uncovered a misconfiguration on the `body-xxs` typography class.

Resolves: [MWPW-137137](https://jira.corp.adobe.com/browse/MWPW-137137)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/sarchibeque/text-legal?martech=off
- After: https://sarchibeque-mwpw-137137--milo--adobecom.hlx.page/drafts/sarchibeque/text-legal?martech=off

### CODE REVIEW

**Review by ryanmparrish — APPROVED** [review]

File: libs/utils/decorate.js [hunk 1]
```diff
@@ -54,20 +54,17 @@ export function decorateBlockText(el, config = ['m', 's', 'm'], type = null) {
...
       }
     }
     const emptyPs = el.querySelectorAll(':scope p:not([class])');
-    if (emptyPs) emptyPs.forEach((p) => { p.classList.add(`body-${config[1]}`); });
-    if (!headings?.length && !emptyPs?.length) {
-      const wrapper = el.querySelector(':scope > div');
-      [...wrapper.children]
-        .filter((child) => child.textContent.trim() !== '')
+    if (emptyPs.length) emptyPs.forEach((p) => p.classList.add(`body-${config[1]}`));
+    else {
```
Comment (chrischrischris) [inline libs/utils/decorate.js:65]: nit: If using `if/else` then wrap the if in {}.

**Review by chrischrischris — COMMENTED** [review]
Just a nit, however I don't see any difference between your branch and main.

See [this website diff](https://pianomister.github.io/diffsite/?url1=https%3A%2F%2Fsarchibeque-mwpw-137137--milo--adobecom.hlx.page%2Fdrafts%2Fsarchibeque%2Ftext-legal%3Fmartech%3Doff&url2=https%3A%2F%2Fmain--milo--adobecom.hlx.page%2Fdrafts%2Fsarchibeque%2Ftext-legal%3Fmartech%3Doff)

Can you provide an example that shows the bug?

**Comment by rgclayton** [comment]
@Sartxi Would mind explain what the difference we should be seeing is? I'm also not seeing a difference between the before and after urls.

File: libs/utils/decorate.js (hunk 1 above)
Comment (Sartxi) [reply libs/utils/decorate.js]: Got it ✅

**Comment by Sartxi** [comment]
@rgclayton @chrischrischris thanks!
Sorry I must have removed the authoring variation that was in the ticket while I was doing my final testing 🤦🏻 
I put it back in there if you can please re-review!

File: libs/styles/styles.css [hunk 2]
```diff
@@ -263,8 +263,8 @@
 }
 
 .body-xxs {
-  font-size: var(--type-heading-xxs-size);
-  line-height: var(--type-body-xs-lh);
+  font-size: var(--type-body-xxs-size);
+  line-height: var(--type-body-xxs-lh);
 }
```
Comment (rgclayton) [inline libs/styles/styles.css:268]: This is the misconfiguration fix?

Comment (Sartxi) [reply libs/styles/styles.css:268]: @rgclayton yes, now that I am using the typography styles for legal variant in text I noticed these were wrong

**Review by rgclayton — APPROVED** [review]

**Comment by rgclayton** [comment]
Working with @skumar09 to update the Nala tests.

**Comment by rgclayton** [comment]
Adding test urls:

- https://main--bacom--adobecom.hlx.page/customer-success-stories/coca-cola-personalization-case-study?milolibs=sarchibeque-mwpw-137137
- https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?milolibs=sarchibeque-mwpw-137137
- https://sarchibeque-mwpw-137137--milo--adobecom.hlx.page/drafts/rclayton/legal-text

**Pull request merged**